arches_search/src/arches_search/AdvancedSearch/types.ts

```diff
@@ -0,0 +1,61 @@
+export interface AdvancedSearchFacet {
+    id: number;
+    arity: number;
+    datatype_id: string;
+    is_orm_template_negated: boolean;
+    label: string;
+    operator: string;
+    orm_template: string;
+    param_formats: string[];
+    sortorder: number;
+}
+
+export interface GraphModel {
```

Comment:
It would be great if these interfaces could be provided and managed by Arches core rather than this app.

Reply:
agreed!

https://github.com/archesproject/arches/issues/12511